test/repro/README.md

```
@@ -0,0 +1,69 @@
+
+```bash
+make clean && PROXYSQL31=1 make debug -j"$(nproc)"      # a DEBUG build is required
+WORKSPACE=$(pwd) INFRA_ID=dev-$USER test/repro/<script>.bash
+```
+
+Both scripts refuse to run against a release build, because the isolated harness
+needs debug-only admin commands.
+
+## COLD_START
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)